scripts/qa_crowbarsetup.sh

```diff
@@ -3656,6 +3661,15 @@ function oncontroller_magnum_service_setup ()
             --disk-format qcow2 --container-format bare --public \
             --property os_distro=fedora-atomic magnum-service-image
     fi
+
+    # create magnum flavors used by tempest
+    if ! openstack flavor show magnum.master; then
+        openstack flavor create --ram 1024 --disk 8 --vcpus 1 magnum.master
+    fi
+
+    if ! openstack flavor show magnum.minion; then
+        openstack flavor create --ram 512 --disk 8 --vcpus 1 magnum.minion
```

Comment:
@bmwiedemann ah.ok. Fixed. Or should I also remove the check (and creation) of ```m1.smaller``` ?. I like it the the current way because the magnum setup code is then in one place.